components/form-builder/app/shared/Toast.tsx

```diff
@@ -85,7 +85,7 @@ export const toast = {
     originalToast.error(<p className="py-2">{message}</p>);
   },
   info: (message: string) => {
-    originalToast.info(<p className="py-2">{message}</p>);
+    originalToast.info(<p className="py-1">{message}</p>);
```

Comment:
Assuming it's just an oversight but why change the padding for only the Info styling?

Reply:
I think that's right, just an oversight. At one point I was trying to "squish" the table cells as close as possible together - it's probably left over from that